includes/admin/helpers/tables/class-fees-table.php

```diff
@@ -174,6 +171,11 @@ public function column_label( $fee ) {
                 ),
                 esc_html__( 'Delete', 'business-directory-plugin' )
             );
+
+			if ( ! $fee->is_enabled_premium() ) {
```

Comment:
I'm not really understanding why this is here, after already checking if these should be added. It looks like this should be removed, and don't unset any actions.

Reply:
I've removed this check, it made sense when payments options was present